WordPress/src/main/res/layout/login_magic_link_sent_fragment.xml

```diff
@@ -0,0 +1,49 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar_login" />
+
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="180dp"
+        android:layout_height="150dp"
+        android:layout_centerHorizontal="true"
+        android:layout_below="@+id/toolbar"
+        android:scaleType="centerInside"
+        android:src="@drawable/login_magiclink_request"/>
```

Comment:
Fixed with 1f003e9! While at it, I added a `contentDescription` to the ImageView (with 48fc977) to lose the yellow warning from AS. Thanks!